src/test/java/uk/gov/hmcts/reform/wataskmanagementapi/domain/entities/CamundaTaskTest.java

```diff
@@ -27,7 +27,8 @@ void should_create_full_object_and_get_values() {
             created,
             dueDate,
             "some-description",
-            "some-owner"
+            "some-owner",
+            "formKey"
         );
 
```

Comment:
I believe we need more assertions here :D as there are a lot more properties in our Task object